Code/src/main/java/nl/utwente/group10/ui/BackendUtils.java

```diff
@@ -31,4 +33,18 @@ public static Type dive(ConstT function, int steps) {
         }
         return function;
     }
+    
+    /**
+     * @return Whether or not t1 and t2 unify with each other.
```

Comment:
The right way to do things is to just apply the argument, then run analyze in a try catch block, and display error information when an exception occurs. The exceptions contain a lot of useful information by the way, including the exact expression object where the error occured.